collections/_components/footer.html

```diff
@@ -0,0 +1,135 @@
+---
+title: Footer
+layout: layout-sidenav
+description: Insert description
+category: mandatory
+react-link: ?path=/docs/components-footer--default-story
+github-link: true
+toc: true
+---
+
+{% capture default-footer %}
+<div class="bg-dark text-light p-4">
```

Comment:
@sophiaaachow 
You can take and copy the entire footer HTML snippets from 
_includes/footer.html